src/GeoCoordinate/GeoPosition.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace System.Device.Location
+{
+	/// <summary>Represents a geographic position consisting of a location and a time stamp.</summary>
+	/// <typeparam name="T"></typeparam>
+	public class GeoPosition<T>
+	{
+		/// <summary>Gets or sets the location of the <see cref="T:System.Device.Location.GeoPosition`1" /> object.</summary>
+		public T Location
+		{
+			get;
+			set;
```

Comment:
Style suggestion but this could be : public T Location { get; set; }

Reply:
We will actually make this class immutable, per JP's suggestion.

Reply:
Well we need to do what System.Device.Location does. If it's not immutable in UWP we shouldn't make our version immutable no?